core/store/src/db.rs

```diff
@@ -30,6 +30,7 @@ pub const LATEST_KNOWN_KEY: &[u8; 12] = b"LATEST_KNOWN";
 pub const LARGEST_TARGET_HEIGHT_KEY: &[u8; 21] = b"LARGEST_TARGET_HEIGHT";
 pub const GENESIS_JSON_HASH_KEY: &[u8; 17] = b"GENESIS_JSON_HASH";
 pub const GENESIS_STATE_ROOTS_KEY: &[u8; 19] = b"GENESIS_STATE_ROOTS";
+pub const CF_STAT_NAMES: [&'static str; 1] = ["rocksdb.total-sst-files-size"];
```

Comment:
Yes, works about 20 times faster and gives the same numbers

Reply:
OK, so firstly, we should be using properties constants from rocksdb crate: `rocksdb::porperties::LIVE_SST_FILES_SIZE`.

Secondly, how long does it take right now to gather those statistics?  There’s also `rocksdb::propertiess::ESTIMATE_LIVE_DATA_SIZE` which sounds like it would be even faster.  Though I couldn’t really find any information how all those metric differ.  But we should be fine tolerating some inaccuracy in the metric if it speeds up collecting the metric.

Reply:
ESTIMATE_LIVE_DATA_SIZE is somehow slower on rpc nodes (by the same 20 times). Right now I am only checking for rpc nodes. The difference in numbers is often 1 percent. 
I kept LIVE_SST_FILES_SIZE and kept the general structure of having it in an array, which is another constant, if in the future we want to have more rocksdb stats for column families.